node/src/accountant/scanners/mod.rs

```diff
@@ -3005,16 +3140,99 @@ mod tests {
             .financial_statistics
             .borrow()
             .total_paid_receivable_wei;
-        let more_money_received_params = more_money_received_params_arc.lock().unwrap();
         assert_eq!(message_opt, None);
         assert_eq!(subject.scan_started_at(), None);
         assert_eq!(total_paid_receivable, 2_222_123_123 + 45_780 + 3_333_345);
+        let more_money_received_params = more_money_received_params_arc.lock().unwrap();
         assert_eq!(*more_money_received_params, vec![(now, receivables)]);
+        let set_through_provided_transaction_params =
+            set_start_block_from_txn_params_arc.lock().unwrap();
+        assert_eq!(
+            *set_through_provided_transaction_params,
+            vec![(7890123, transaction_id)]
+        );
+        let commit_params = commit_params_arc.lock().unwrap();
+        assert_eq!(*commit_params, vec![()]);
         TestLogHandler::new().exists_log_matching(
             "INFO: receivable_scanner_handles_received_payments_message: The Receivables scan ended in \\d+ms.",
         );
     }
 
+    #[test]
+    #[should_panic(expected = "Attempt to set new start block to 7890123 failed due to: \
+    DatabaseError(\"Fatigue\")")]
+    fn received_transactions_processed_but_start_block_setting_fails() {
```

Comment:
This one too, it should be a log instead of a panic.

Reply:
Nope. Shouldn't.